assets/src/edit-story/app/api/test/apiProvider.js

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * WordPress dependencies
+ */
+
+/**
+ * External dependencies
+ */
+import { act, renderHook } from '@testing-library/react-hooks';
+
+jest.mock('@wordpress/api-fetch');
+import apiFetch from '@wordpress/api-fetch';
```

Comment:
The import can still be added to the 'WordPress dependencies' section.

Reply:
Done.